index.js

```diff
@@ -172,7 +165,6 @@ module.exports = function makePlugin(opts) {
       });
 
       return () => {
-        clientPeer.close();
```

Comment:
Does hyperswarm not provide any API to close? I think it's an important part

Reply:
It does not.  At least nothing documented not recommended in examples, as far as I've been able to find.

Reply:
It has `destroy` https://github.com/hyperswarm/hyperswarm/blob/924f56ae2781f1c605916e8e070f474ac7fff0f0/swarm.js#L302

In general it's a good idea to dig into source code if you can't find examples or docs. Mafintosh's code is very decently structured.

Reply:
I'm not generally a fan of using undocumented APIs, but if you know Mafintosh is consistent with them, that's fine.  See latest commit.